README.md

```diff
@@ -69,5 +69,26 @@ In that case, you can use the `nuke-testrunner`
 rye run nuke-testrunner -n <path_to_nuke> -t ./tests
 ```
 
```

Comment:
Would be good to add a small header here, 3x hashtag I guess

Reply:
I moved the section. It's better suited for the how to use section.